action.yml

```
@@ -105,7 +105,7 @@ runs:
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
         fi
 
-        echo "::set-output name=image::$ECR_REGISTRY/$IMAGE_NAME:$IMAGE_TAG"
+        echo "image=$ECR_REGISTRY/$IMAGE_NAME:$IMAGE_TAG" >> $GITHUB_STATE
```

Segun la docu seria asi: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Antes estabamos seteandolo al output del step y no del state, no?
```suggestion
        echo "image=$ECR_REGISTRY/$IMAGE_NAME:$IMAGE_TAG" >> $GITHUB_OUTPUT
```